README.md

```diff
@@ -45,6 +45,7 @@ http://0.0.0.0:5898/metrics
 | fileGlobPattern | Wildcard pattern of gc log file path, multiple paths separate with commas(,) |
 | idleTimeout     | Milliseconds before closing idle(no update) files, default is 1 hour         |
 | watchInterval   | Time interval for scanning matching files (ms)                               |
+| readInterval    | Time to sleep between files reading (ms)                                     |
```

Comment:
With the "release" of the files between reading, rotation problem is less probable, but not 0, decreasing the "lock" time lower again the probability.  
This exporter is only pulled every minute by Prometheus in my project, I thougth that a reading of every second is overkill. for example in my configuration I set it to 20s.

And because there are a watchInterval, which give the same feature for registered files.

So I though that it can be a good idea to put it in configuration, for other users.